internal/agent/agent.go

```diff
@@ -276,6 +280,15 @@ func (h *ApplyHandlerv1) preflightCheck(w http.ResponseWriter, r *http.Request)
 func (h *ApplyHandlerv1) ProcessRequest(lbcfg *model.LoadBalancer) (int, string) {
 	klog.V(1).Infof("received config: %#v", lbcfg)
 
+	err := validate.Struct(lbcfg)
+	if err != nil {
+		for _, e := range err.(validator.ValidationErrors) {
+			klog.Errorf("%s\n", e.Error())
+			klog.V(1).Infof("%#v\n", e)
+		}
+		return 443, err.Error() // Bad Request
```

Comment:
443? Is that supposed to be an HTTP status code? Where is it defined? Did you mean 400 ("Bad Request")?

Reply:
I did, and I have no idea where that 443 came from :D